instrumentation/opentelemetry-instrumentation-confluent-kafka/src/opentelemetry/instrumentation/confluent_kafka/__init__.py

```diff
@@ -356,6 +359,9 @@ def wrap_produce(func, instance, tracer, args, kwargs):
 
     @staticmethod
     def wrap_poll(func, instance, tracer, args, kwargs):
+        if context.get_value(_SUPPRESS_INSTRUMENTATION_KEY):
```

Comment:
We are trying to use the [utils](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/opentelemetry-instrumentation/src/opentelemetry/instrumentation/utils.py#L167) function for this check instead.

Reply:
@oleksii-valuiskyi just in case you need an example on how do that change: https://github.com/open-telemetry/opentelemetry-python-contrib/pull/2590/files#diff-48337575ba4313bb370c7fecba131f2fdf861ec0748d75bad061e04ce81ff2a8R195